src/views/PlanTargets.vue

```diff
@@ -87,13 +97,34 @@ export default {
       targlist: '',
       lat1: '',
       lon1: '',
-      dateobs: new Date(Math.round(new Date().getTime() / 1800000) * 1800000),
+      customobservatoryoffset: '',
```

Comment:
The browser console is warning about this value not matching the expected type (should be a number instead of a string). If you change this initial value to 0, it goes away.